public/js/components/stories/Frames.js

```diff
@@ -1,46 +1,26 @@
 "use strict";
 
 const React = require("react");
-const { DOM: dom, createElement } = React;
-const { Provider } = require("react-redux");
-const { Map, fromJS } = require("immutable");
+const { DOM: dom, createElement, createFactory } = React;
+const { renderComponent, storiesOf } = require("./utils");
 
-const { createStore } = require("./utils");
+const Frames = require("../Frames");
 
-const { storiesOf } = require("@kadira/storybook");
-const Frames = React.createFactory(require("../Frames"));
-const fixtures = require("../../test/fixtures");
+const style = {
+  width: "300px",
+  margin: "auto",
+  paddingTop: "100px" };
 
-function getData(fixtureName) {
-  const fixture = fixtures[fixtureName];
-  const frames = fixture.pause.frames;
-  return {
-    pause: Map({ frames }),
-    sources: fromJS({ sources: fixture.sources.sources })
-  };
```

Comment:
`getData` and `renderContainer` are both gone in favor of a shared `renderComponent` function